# represent the object belief state
uint32 obj_id
geometry_msgs/Pose pose
VoxelGridBool con_volume  # conservative volume
VoxelGridBool opt_volume  # optimistic volume
